mobify/progressive-web-scaffold
# Cart stepper functionality

This PR fixes the stepper on the cart so it correctly updates the qty of an item.

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1110
 **Linked PRs**: n/a

## Changes
- Added onChange handler to stepper for cart items
- Updated updateItemQuantity action to not use FormData
- Moved getCart call into updateItemQuantity so we don't need to call both functions to update the qty

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart
- Increase/decrease the qty for the item
- The cart should update correctly
